SpecCheck.py

```diff
@@ -444,6 +445,10 @@ def check_spec(self, pkg, spec_file, spec_lines=None):
                                          'comparison-operator-in-deptoken',
                                          conf)
 
+            if current_section in ('post', 'postun'):
+                if lconfig_regex.search(line):
```

Comment:
What's the point of up streaming this patch? The macro isn't defined anymore, we should just drop it

Reply:
That macro is defined in 24 packages in tumbleweed.